x-pack/docs/en/security/limitations.asciidoc

```diff
@@ -79,9 +79,15 @@ including the following queries:
 ** `percolate` query
 * If suggesters are specified and document level security is enabled, the specified suggesters are ignored.
 * A search request cannot be profiled if document level security is enabled.
-* The <<search-terms-enum,terms enum API>> does not return terms if document 
+* The <<search-terms-enum,terms enum API>> does not return terms if document
 level security is enabled.
 
+While document-level security prevents users from seeing documents that they do not have access to,
+it is possible to craft search requests in order to discover aggregate information about the entire
+index. A user whose access to an index is restricted via document-level security could still learn
+about fields names and terms that only exist in inaccessible documents and count how many inaccessible
+documents containing a given term. But no more than that.
```

Comment:
```suggestion
NOTE: While document-level security prevents users from viewing restricted documents,
it's still possible to write search requests that return aggregate information about the
entire index. A user whose access is restricted to specific documents in an index could
still learn about field names and terms that only exist in inaccessible
documents, and count how many inaccessible documents contain a given term. 
```

Reply:
@ywangd, are there security implications here? If a user can count how many inaccessible documents can contain specific field names and terms, can they do anything? I'm just curious how important it is to tell users about this limitation.